src/renderer/components/AppUpdate/AppUpdate.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+
+import { Grid } from 'antd'
+import { useIntl } from 'react-intl'
+
+import * as Styled from './AppUpdate.styles'
+
+export type AppUpdateModalProps =
+  | {
+      isOpen: true
+      type: 'success'
+      goToUpdates: () => void
+      version: string
+      close: () => void
+    }
+  | {
+      isOpen: true
+      type: 'fail'
+      close: () => void
+      message: string
+    }
+  | {
+      isOpen: false
+    }
+
+export const AppUpdate: React.FC<AppUpdateModalProps> = (props) => {
+  const intl = useIntl()
+  const isDesktopView = Grid.useBreakpoint()?.lg ?? false
+
+  if (props.isOpen && props.type === 'success') {
```

Comment:
Pls destruct values from `props` before (as we do it in other React components).